README.md

```diff
@@ -1,2 +1,55 @@
 # storage-postgres
-A tailored version of PostgreSQL used as a Storage Plugin for FogLAMP
+This repository provides a tailored version of PostgreSQL used as a Storage Plugin for FogLAMP.
+
+Download a tarball with the prepared version of PostgreSQL available here:
+* For Intel [x86_64](https://s3.amazonaws.com/foglamp/plugins/storage/postgres/pgsql-foglamp-9.6_201608131-x86_64.tgz).
```

Comment:
Must we used the prepared version? It would be better if we could use a standard tarball from postgres.

Reply:
This is a tailored version with less executables. We are going to customize even more in the future. 
If people want to use the original PG, they are free to do so...

Reply:
We should have a script or instructions (so that we can script) creation of a tailored version. That way a person can create a debian from a a default postgres tarball.

Reply:
yes we should, but this is a very low priority compared to all the work to do...